### DISCUSSION TITLE
Windows packaging IPA results in Invalid Bundle

### DISCUSSION THREAD

**Discussion by marchbold** [airsdk/Adobe-Runtime-Support#802] [Q&A]
Just wondering if anyone has encountered this issue before:

```
ERROR ITMS-90208: "Invalid Bundle. The bundle APPLICATION.app 
    does not support the minimum OS Version specified in the Info.plist."
```

The error occurs when verifying the IPA with appstore connect and only happens when building on a windows machine (macOS build is fine).

- I have tried with several recent versions of AIR (including 444).
- Tried several values for `MinimumOSVersion` flag in the info additions and removing completely.
- Tried with and without the `ios_version_min` option (in an ANE)

**Comment by ajwfrost** [comment]
If you unzip the IPA file, on a Mac, can you check
1) in the `Payload/APPLICATION.app/Info.plist` file, what's the value under `MinimumOSVersion`?
2) if you run `otool -l Payload/APPLICATION.app/APPLICATION > output.txt` and look in the output.txt file for a setting `LC_VERSION_MIN_IPHONEOS` - what does that give?

e.g. for me:
```
        <key>MinimumOSVersion</key>
        <string>9.0</string>
```
and
```
Load command 9
      cmd LC_VERSION_MIN_IPHONEOS
  cmdsize 16
  version 9.0
      sdk 14.1
```

**Reply by marchbold** [reply]
There is no `LC_VERSION_MIN_IPHONEOS` entry, there's this one:

```
Load command 9
      cmd LC_BUILD_VERSION
  cmdsize 24
 platform 2
    minos 12.0
      sdk 14.1
```

The info plist has the version i specified in the info additions 

```
        <key>MinimumOSVersion</key>
        <string>10.0</string>
```

**Reply by marchbold** [reply]
The one built on macOS has the `LC_VERSION_MIN_IPHONEOS` entry but not the `LC_BUILD_VERSION`

**Reply by ajwfrost** [reply]
On the macOS one -> is that a simulator build?

Thanks .. so it seems that something is making the binary have the minimum SDK version of 12.0... a solution to get it working would be to update the info.plist minimum version to 12.0 to match it! but it would presumably be useful to find out why the 12.0 value is embedded in the binary to start with...  looking at the code it should be the value in the InfoAdditions that sets this up (and if you didn't have any value there, then the Info.plist file should have a '9' for the minimum OS version I think?)

We don't have the debug output set up for this part of ADT, which is perhaps something that needs to be updated!

thanks

**Reply by marchbold** [reply]
No both were app store distribution builds. Setting it to 12.0 does not resolve the error, we still get the same Invalid bundle message. (I'll double check that in the morning though)

**Reply by marchbold** [reply]
Okay so I increased the minimum version to 12 and added `arm64` to the `UIRequiredDeviceCapabilities` and it has verified now. 

However not really a solution as it should work with 9 as the minimum and does when packaged on macos.
